### ISSUE TITLE
1.12.65: an undefined IP address should be valid for a virtual host in a cluster

### ISSUE THREAD

**Issue by jouvin** [quattor/aquilon#127]
Currently, it is required that an address is declared (either directly or indirectly) for any host. It is not always appropriate, in particular for virtual host hosted by a cluster representing a cloud where the actual address allocation will occur according to the cloud policy defined. It would be good to allow an undefined IP address in this case. It would be up to the user not to configure the network with Quattor in this situation (for example, at LAL we disable `ncm-network` in virtual machines).

**Comment by ned21** [comment]
We might have an upcoming requirement for this too, although we probably want to generalise it to be any host not just a VM. I wasn't aware that the standard schemas allowed it, but if you are using it that way I guess that means AII, _et al_. will work?

**Comment by jouvin** [comment]
@ned21 no, we are not using it that way and I am not sure the schema allows it... I need to check. What we do currently, as we always did with VM, is that we disable ncm-network. I think that if ncm-network is disabled, we don't really care if an Ip address is defined, Even for AII it should work if a DHCP address is obtained. But in our case, we don't care about AII with VMs as we don't install them this way.
